test/cartesian/ErrorBar.spec.tsx

```diff
@@ -103,4 +116,90 @@ describe('<ErrorBar />', () => {
 
     expect(container.querySelectorAll('.recharts-errorBar')).toHaveLength(10);
   });
+
+  test('Renders Error Bars with animation', () => {
+    const { container } = render(
+      <LineChart data={barData} width={500} height={500}>
+        <Line isAnimationActive dataKey="uv">
+          <ErrorBar dataKey="uvError" />
+        </Line>
+      </LineChart>,
+    );
+
+    assertErrorBars(container, 0);
+
+    setTimeout(() => {
```

Comment:
If we use `setTimeout` in a test then failure from here will be reported in some completely different test in completely different file and mightily confuse everyone. Please use fake timers instead: https://vitest.dev/guide/mocking.html#timers

Reply:
thanks for the suggestion, tried to follow the document using `useFakeTimers` and `advanceTimersByTime` but failed to pass the test, could you give me any guidance on this?

```javascript
beforeAll(() => {
  vi.useFakeTimers();
});

afterAll(() => {
  vi.useRealTimers();
});

test('Renders Error Bars with animation', () => {
    const { container } = render(
      <LineChart data={barData} width={500} height={500}>
        <Line isAnimationActive dataKey="uv">
          <ErrorBar dataKey="uvError" />
        </Line>
      </LineChart>,
    );
      
    vi.advanceTimersByTime(200);
    assertErrorBars(container, 4);
    assertAnimationStyles(container, 'scale(1, 1)', 'transform 200ms ease-in-out 0s');
});
```